### ISSUE TITLE
Воспроизведение перевода по факту начала проигрывания видео

### ISSUE THREAD

**Issue by GennPen** [ilyhalight/voice-over-translation#639]
### Describe your feature / Опишите ваше предложение

Есть ли возможность добавить опцию начала воспроизведения перевода по факту начала проигрывания видео?

Многие видео начинают воспроизводиться не сразу (связываю это скорее всего с блокировками РКН) и пока видео ждет секунд 5-7 начала воспроизведения, перевод уже начинает воспроизводиться. И потом, когда уже видео фактически начинает воспроизводиться перевод начинает воспроизводиться заново.
Не сильно критично, но иногда раздражает.

**Comment by SashaXser** [comment]
> ### Describe your feature / Опишите ваше предложение
> 
> Есть ли возможность добавить опцию начала воспроизведения перевода по факту начала проигрывания видео?
> 
> Многие видео начинают воспроизводиться не сразу (связываю это скорее всего с блокировками РКН) и пока видео ждет секунд 5-7 начала воспроизведения, перевод уже начинает воспроизводиться. И потом, когда уже видео фактически начинает воспроизводиться перевод начинает воспроизводиться заново. Не сильно критично, но иногда раздражает.

Можешь показать как это выглядит?

**Comment by GennPen** [comment]
> Можешь показать как это выглядит?

Примерно так: https://disk.yandex.ru/i/RgClQutIRx7ACg

**Comment by SashaXser** [comment]
> > Можешь показать как это выглядит?
> 
> Примерно так: https://disk.yandex.ru/i/RgClQutIRx7ACg

Хм, а это случайно не задержка от ютуба когда у тебя блокировщик рекламы включен?

**Comment by GennPen** [comment]
> Хм, а это случайно не задержка от ютуба когда у тебя блокировщик рекламы включен?

Нет, это задержка загрузки. Отключение блокировщика не изменяет ситуации. Через VPN все сразу начинает нормально воспроизводиться.

**Comment by SashaXser** [comment]
> > Хм, а это случайно не задержка от ютуба когда у тебя блокировщик рекламы включен?
> 
> Нет, это задержка загрузки. Отключение блокировщика не изменяет ситуации, ровно как и в режиме инкогнито без аддонов. Через VPN все сразу начинает нормально воспроизводиться.

> > Хм, а это случайно не задержка от ютуба когда у тебя блокировщик рекламы включен?
> 


Попробуй эту версию [скрипта](https://github.com/SashaXser/voice-over-translation/tree/test/dist)

**Comment by GennPen** [comment]
> Попробуй эту версию [скрипта](https://github.com/SashaXser/voice-over-translation/tree/test/dist)

Без изменений. Также начинает воспроизведение перевода.

**Comment by FiLinX** [comment]
[GennPen](https://github.com/GennPen)  вообще, лучшим решением будет применение функции запрета автоплея!
автоплей - это очень плохо..
отключение его, можно найти в почти любом расширении для ютуба, например в [Enhancer for YouTube™](https://www.mrfdev.com/enhancer-for-youtube)

**Comment by ilyhalight** [comment]
в последней версии не должно быть таких проблем
